src/utils/combo/filter.ts

```diff
@@ -17,9 +17,22 @@ export interface ComboFilterSettings {
   wobbleThreshold: number; // The number of pummels before it's considered a wobble
   chainGrabThreshold: number; // proportion of up throw / pummels to other moves to be considered a chain grab
   perCharacterMinComboPercent: { [characterId: number]: number };
+  includesComboSequence: ComboSequenceFilter; // List of moveIDs to appear in sequence
   fuzzyNameTagMatching: boolean;
 }
 
+export enum ComboSequenceFilterMode {
+  "include",
+  "start",
+  "end",
+  "exact",
+}
+
+export type ComboSequenceFilter = {
+  sequence: number[];
+  mode?: ComboSequenceFilterMode;
+};
```

Comment:
```suggestion
export type ComboSequenceFilter = {
  sequence: number[];
  mode?: "include" | "start" | "end" | "exact",
};
```
What are your thoughts on just inlining this one? I'm not sure we really need an exported enum for this.

Or alternatively:
```suggestion
// I'm not super convinced we need to export this type
type ComboSequenceFilterMode = "include" | "start" | "end" | "exact";

export type ComboSequenceFilter = {
  sequence: number[];
  mode?: ComboSequenceFilterMode;
};
```